Eliminate dependency on intel-mpi-rt-mic which in turn brings in intel-mpi-rt.
author: Bart Oldeman (McGill HPC) 
--- pset/mediaconfig.xml.orig      2016-10-20 09:02:57.000000000 +0000
+++ pset/mediaconfig.xml   2016-11-11 20:09:08.084892189 +0000
@@ -1551,7 +1551,7 @@
       <MapFileDir>${BUNDLE_MAPFILE_DIR}</MapFileDir>
       <ChklicIA32>${BUNDLE_CHKLIC_IA32}</ChklicIA32>
     </Component>
-    <Component depend="1.3;33.111;33.112;1.432;118.3" id="114" invisible="0" mandatory="0" platform="INTEL64">
+    <Component depend="1.3;33.111;33.112;1.432" id="114" invisible="0" mandatory="0" platform="INTEL64">
       <Name>Intel Fortran Compiler for Intel(R) 64</Name>
       <Description>Intel(R) Fortran Compiler 17.0 Update 1</Description>
       <Name locale="ja_JP.UTF-8">インテル(R) Fortran コンパイラー (インテル(R) 64)</Name>
